packages/plugin-style-inhibitor/src/index.js

```diff
@@ -0,0 +1,28 @@
+import { hookMethod } from '@merkur/core';
+import inhibitOuterStyles from './styleInhibitor';
+
+function styleInhibitorPlugin() {
+  return {
+    async setup(widget) {
+      return widget;
+    },
+    async create(widget) {
+      hookMethod(widget, 'mount', mountHook);
+
+      return widget;
+    },
+  };
+}
+
+async function mountHook(widget, originalMount, ...rest) {
+  const result = await originalMount(...rest);
+
+  if (typeof window === 'object') {
+    const container = document.querySelector(widget.props.containerSelector);
```

Comment:
Since merkur 0.24.0>=, `widget.props.containerSelector` in props is deprecated and was moved to `widget.containerSelector`. 

This should be fixed or left with fallback support for old definition, if you still need to use this with older merkur widget integrations.

Also do you plan to support slots? (each slot can have different root container)